src/SecureElementReader.App/ViewModels/MenuViewModel.cs

```diff
@@ -1,37 +1,69 @@
-﻿using ReactiveUI;
+﻿using Avalonia.Markup.Xaml.MarkupExtensions;
+using ReactiveUI;
 using SecureElementReader.App.Interfaces;
+using SecureElementReader.App.Models.Configurations;
 using SecureElementReader.App.ViewModels.Interfaces;
 using SecureElementReader.App.ViewModels.Services;
 using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Input;
 
 namespace SecureElementReader.App.ViewModels
 {
     public class MenuViewModel : ViewModelBase, IMenuViewModel
     {
-        private readonly IDialogService dialogService;
+        private readonly SelectedLanguageConfiguration _configuration;
+        private readonly IDialogService _dialogService;
+        private readonly ILocalizationService _localizationService;
+
 
         public ICommand ExitCommand { get; }
 
         public ICommand AboutCommand { get; }
 
         public MenuViewModel(
             IApplicationCloser applicationCloser,
-            IDialogService dialogService)
+            IDialogService dialogService,
+            ILocalizationService localizationService,
+            SelectedLanguageConfiguration configuration)
         {
-            this.dialogService = dialogService;
+            this._dialogService = dialogService;
+            _localizationService = localizationService;
+            _configuration = configuration;
 
             ExitCommand = ReactiveCommand.Create(applicationCloser.CloseApp);
             AboutCommand = ReactiveCommand.CreateFromTask(ShowAboutDialogAsync);
+
         }
 
         private Task ShowAboutDialogAsync()
         {
-            return dialogService.ShowDialogAsync(nameof(AboutDialogViewModel));
+            return _dialogService.ShowDialogAsync(nameof(AboutDialogViewModel));
+        }
+
+        public void Translate(string targetLanguage)
+        {
+            var translations = App.Current.Resources.MergedDictionaries.OfType<ResourceInclude>().FirstOrDefault(x => x.Source?.OriginalString?.Contains("Translations.EN.axaml") ?? false);
+
+            if (translations != null)
+                App.Current.Resources.MergedDictionaries.Remove(translations);
```

Comment:
Always use curly braces for code block after if, even if it there is only one line of code